packages/dialogs/src/RPA/Dialogs/library.py

```diff
@@ -696,6 +697,53 @@ def add_drop_down(
 
         self.add_element(element)
 
+    @keyword("Add date-picker", tags=["input"])
```

Comment:
I think the name for the  keyword should actually be **Add Date Input** to follow naming convention within current library implementation.